lhotse/recipes/librimix_mini.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import shutil
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, Optional, Union
+from zipfile import ZipFile
+
+from lhotse import fix_manifests, validate_recordings_and_supervisions
+from lhotse.audio import AudioSource, Recording, RecordingSet
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike, Seconds, resumable_download
+
+
+def download_librimix_mini(
+    target_dir: Pathlike = ".",
+    force_download: Optional[bool] = False,
+    url: Optional[str] = "https://zenodo.org/record/3871592/files/MiniLibriMix.zip",
+) -> Path:
+    target_dir = Path(target_dir)
+    target_dir.mkdir(parents=True, exist_ok=True)
+    zip_path = target_dir / "MiniLibriMix.zip"
+    unzipped_dir = target_dir / "MiniLibriMix"
+    completed_detector = unzipped_dir / ".completed"
+    if completed_detector.is_file():
+        logging.info(f"Skipping {zip_path} because {completed_detector} exists.")
+        return unzipped_dir
+    resumable_download(
+        url,
+        filename=zip_path,
+        force_download=force_download,
+    )
+    shutil.rmtree(unzipped_dir, ignore_errors=True)
+    with ZipFile(zip_path) as zf:
+        zf.extractall(path=target_dir)
+    completed_detector.touch()
+    return unzipped_dir
+
+
+def prepare_librimix_mini(
+    librimix_csv: Pathlike,
+    output_dir: Optional[Pathlike] = None,
+    with_precomputed_mixtures: bool = False,
+    sampling_rate: int = 16000,
+    min_segment_seconds: Seconds = 3.0,
+) -> Dict[str, Dict[str, Union[RecordingSet, SupervisionSet]]]:
+    import pandas as pd
```

Comment:
pandas does not seem to be in the dependencies of lhotse:
https://github.com/lhotse-speech/lhotse/blob/master/setup.py#L150

(up on Piotr to decide, if pandas is ok)